director/scripts/test/kinematic_test.py

```diff
@@ -30,21 +34,36 @@ def __init__(self, parent = None):
     def handle_thread(self):
         self.channel = "NETWORK_CASSIE_STATE_DISPATCHER"
         self.lcm = lcm.LCM()
+        self.prev_loc = {}
+        self.json_file = sys.argv[3];
+        self.duration = {}
```

Comment:
Fields like these need to be better organized (e.g. by data type). See comment below also on JSON parsing.